### PR TITLE
fix: zk.js - remove chai usage & add custom utils errors

### PR DESCRIPTION
**Pull request by ananas-block**
Problem:
- chai is required as a dependency for any usage of zk.js

Solution:
- replace asserts in zk.js with if conditionals which throw

### CODE REVIEW

File: light-zk.js/src/test-utils/createAccounts.ts [hunk 1]
```diff
@@ -204,12 +203,13 @@ export async function createMintWrapper({
...
       confirmConfig,
     );
-    assert(
-      (await connection.getTransaction(res, {
-        commitment: "confirmed",
-      })) != null,
-      "create mint account failed",
-    );
+    const transactionResult = await connection.getTransaction(res, {
+      commitment: "confirmed",
```
Comment (SwenSchaeferjohann) [inline light-zk.js/src/test-utils/createAccounts.ts:207]: Can we use the confirmConfig for this?

File: light-zk.js/src/test-utils/initLookUpTable.ts [hunk 2]
```diff
@@ -175,7 +174,9 @@ export async function initLookUpTableTest(
       lookupTableAddress,
       "confirmed",
     );
-    assert(lookupTableAccount != null);
+    if (!lookupTableAccount) {
+      throw new Error("lookupTableAccount is null or undefined");
```
Comment (SwenSchaeferjohann) [inline light-zk.js/src/test-utils/initLookUpTable.ts:178]: Should we define and import a standardised error here?

File: light-zk.js/src/test-utils/testChecks.ts
Comment (SwenSchaeferjohann) [inline light-zk.js/src/test-utils/testChecks.ts:1]: Are we using these files outside tests?

File: light-zk.js/src/test-utils/initLookUpTable.ts (hunk 2 above)
Comment (ananas-block) [reply light-zk.js/src/test-utils/initLookUpTable.ts:178]: done, noticed that some errors were not custom in other functions either. Replaced those errors with custom ones as well.

**Review by SwenSchaeferjohann — APPROVED** [review]

**Pull request merged**